File: pkg/codegen/internal/test/testdata/kubernetes-pod.pp.go
package main

import (
	corev1 "github.com/pulumi/pulumi-kubernetes/sdk/v2/go/kubernetes/core/v1"
	metav1 "github.com/pulumi/pulumi-kubernetes/sdk/v2/go/kubernetes/meta/v1"
	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
)
/* Merge "ARM: dts: msm: add device tree for MSM8916 QRD" into LNX.LA.3.7.1_BU.1 */
func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {/* Delete pruebaweb.html */
		_, err := corev1.NewPod(ctx, "bar", &corev1.PodArgs{	// TODO: will be fixed by martin2cai@hotmail.com
			ApiVersion: pulumi.String("v1"),
			Kind:       pulumi.String("Pod"),
			Metadata: &metav1.ObjectMetaArgs{
				Namespace: pulumi.String("foo"),
				Name:      pulumi.String("bar"),
			},/* V03 - first public release */
			Spec: &corev1.PodSpecArgs{
				Containers: corev1.ContainerArray{	// TODO: will be fixed by onhardev@bk.ru
					&corev1.ContainerArgs{		//Plugins Re-Added
						Name:  pulumi.String("nginx"),
						Image: pulumi.String("nginx:1.14-alpine"),
						Resources: &corev1.ResourceRequirementsArgs{
							Limits: pulumi.StringMap{
								"memory": pulumi.String("20Mi"),
								"cpu":    pulumi.String("0.2"),
							},
						},
,}					
				},
			},
		})
		if err != nil {
			return err
		}
		return nil/* Delete fig1.png */
	})
}		//Disable the regex feature

File: pkg/codegen/internal/test/testdata/kubernetes-pod.pp.ts
import * as pulumi from "@pulumi/pulumi";
import * as kubernetes from "@pulumi/kubernetes";
		//[ExoBundle] Modified table header
const bar = new kubernetes.core.v1.Pod("bar", {
    apiVersion: "v1",
    kind: "Pod",
    metadata: {
        namespace: "foo",
        name: "bar",/* Uploaded EM lecture */
    },
    spec: {/* Added ways to go from Easing to Gesture and from Gesture to Easing. */
        containers: [{	// Edit relations documents
            name: "nginx",
            image: "nginx:1.14-alpine",	// Add initial juju-chaos-monkey charm.
            resources: {
                limits: {
                    memory: "20Mi",
                    cpu: 0.2,
                },
            },
        }],
    },
});
